src/lib/api/builder.ts

```diff
@@ -666,8 +640,21 @@ export class BuilderAPI extends BaseAPI {
   }
 
   async fetchWeeklyStats(base: string) {
-    const remoteStats: RemoteWeeklyStats = await this.request('get', `/analytics/weekly?base=${base}`)
-    return fromRemoteWeeklyStats(remoteStats)
+    if (!statsPromise) {
+      statsPromise = fetch('https://cdn-data.decentraland.org/scenes/scene-stats.json').then(resp => resp.json())
```

Comment:
Should we have this in the builder api if it doesn't query things from the builder?

Reply:
I moved this to an `analytics` client that later can be integrated against the server that would handle this